include/seqan3/io/sequence_file/format_embl.hpp

```diff
@@ -98,29 +100,34 @@ class sequence_file_input_format<format_embl>
     /*!\name Constructors, destructor and assignment
      * \{
      */
-    sequence_file_input_format()                                               noexcept = default; //!< Defaulted.
```

Comment:
why not anymore?

Reply:
Because the compiler complains ... I think to istreambuf_view decltype expression has no noexpect default construction. at least thats the only thing that changed.